docs/integrations.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: default
+title: Integrations
+has_children: false
+has_toc: false
+nav_order: 9
+---
+
+# Integration with SmartClean Matrix
+_Integrations allow you to set up access to data and events from our platform in a standard way._
+
+## Integration mechanisms supported
+1. Receive data and events from our platform on your web service.
```

Comment:
Push Based - Receive data and events from our platform on your service host over https.

Reply:
Resolved by commit:
https://github.com/smartclean/smartclean.github.io/pull/79/commits/46a827ac0238774caae60421357e894dec515a7d